cloud-dataplane/cloud-dataplane.yaml

```diff
@@ -4976,14 +4976,14 @@ security:
   - auth0: []
 servers:
   - description: Data Plane API
-    url: 'https://{dataplane_api_url}.cloud.redpanda.com'
+    url: '{dataplane_api_url}'
     variables:
       dataplane_api_url:
-        default: "dataplane_api.url"
+        default: https://{dataplane_api_url}
         description: |-
           Find the Data Plane API base URL of a cluster by calling the Get Cluster endpoint of the Control Plane API. The dataplane_api.url field is returned in the response body.<br><br>
-          Example (Dedicated): "https://api-a4cb21.ck09mi9c4vs17hng9gig.fmc.prd.cloud.redpanda.com"<br>
-          Example (BYOC): "https://api-a4cb21.ck09mi9c4vs17hng9gig.byoc.prd.cloud.redpanda.com"
+          					Example (Dedicated): "https://api-a4cb21.ck09mi9c4vs17hng9gig.fmc.prd.cloud.redpanda.com"<br>
+          					Example (BYOC): "https://api-a4cb21.ck09mi9c4vs17hng9gig.byoc.prd.cloud.redpanda.com"
```

Comment:
I think we should always put BYOC before Dedicated

Reply:
This is lifted from the source in cloudv2, but we can change it here and then backport.